app/javascript/components/workflows/professor/exams/admin.tsx

```diff
@@ -292,40 +315,57 @@ const VersionInfo: React.FC<{
 const ShowVersion: React.FC<{
   version: Version;
   examName: string;
+  refresh: () => void;
 }> = (props) => {
   const {
     version,
     examName,
+    refresh,
   } = props;
   const { examId } = useParams();
+  const { alert } = useContext(AlertContext);
   const [preview, setPreview] = useState(false);
   return (
     <>
-      <InputGroup>
-        <h3 className="flex-grow-1">{version.name}</h3>
-        <InputGroup.Append>
-          <ButtonGroup>
-            <Button
-              variant="info"
-            >
-              Grade
```

Comment:
Where did this button go?  It seems like this will still be needed... ;-)

Reply:
Yes, but we don't have anything for grading yet, and we can add this button once we know when to show it.

Reply:
Ok... Not that we're likely to forget this, but would've wanted a TODO reminder...  (Not worth blocking this PR for it right now, though)